instructor/templates/instructor/base.html

```diff
@@ -0,0 +1,188 @@
+        <a class="scb_f_login" href="#view=assignments" role="presentation" aria-label="Sign In link">
+            <span class="scb_s_login_status" aria-hidden="true">SIGN OUT</span>
+        </a>
+    </span>
+</div>
+
+<div class="scb_s_instructor_account_title">Instructor Account: Problem Set Builder</div>
+
+<div class="scb_s_assignment_step" role="menu" xmlns="http://www.w3.org/1999/html">
+    <a class="scb_s_assignment_step_link scb_f_assignments_step_link scb_s_assignment_step_link_active"
+        href="{% url "common_assignments" %}" aria-label="Assignments" role="menuitem">
+        <div class="scb_s_assignment_step_wrapper" style="position: absolute; left: 29px; bottom: 20px;"
+         aria-hidden="true">
+        <div class="scb_s_assignments_link_img_active" role="presentation"></div>
+        DASHBOARD
+        </div>
+    </a>
+</div>
+
+{% block body %}
+    {% if messages %}
+        <div>
+            <strong>Messages:</strong>
+            <ul>
+                {% for message in messages %}
+                    <li>{{ message }}</li>
+                {% endfor %}
+            </ul>
+        </div>
+    {% endif %}
+
+{#    <div>#}
+{#        <strong>Menu:</strong>#}
+{#        <ul>#}
+{#            {% if user.is_authenticated %}#}
+{#                <li><a href="{% url 'account_email' %}">Change E-mail</a></li>#}
+{#                <li><a href="{% url 'account_logout' %}">Sign Out</a></li>#}
+{#            {% else %}#}
+{#                <li><a href="{% url 'account_login' %}">Sign In</a></li>#}
+{#                <li><a href="{% url 'account_signup' %}">Sign Up</a></li>#}
+{#            {% endif %}#}
+{#        </ul>#}
+{#    </div>#}
+
+    {% block content %}
+    {% endblock %}
+{% endblock %}
+{% block extra_body %}
+{% endblock %}
+
+
+<div class="scb_s_footer" role="footer"><a href="http://web.mit.edu/" role="link"><img
+        class="scb_s_instructor_homepage_footer_logo" src="images/homepage/mit_logo.png" alt="MIT Logo"></a><img
+        class="scb_s_instructor_homepage_footer_divider" role="presentation"
+        src="images/homepage/small_divider.png" alt=""><a href="http://star.mit.edu/about.html" role="link"
+                                                          aria-label="About Star" target="_blank"><span
+        aria-hidden="true" class="scb_s_instructor_homepage_footer_about" src="images/homepage/about_star.png">About STAR</span></a><img
+        class="scb_s_instructor_homepage_footer_divider" role="presentation"
+        src="images/homepage/small_divider.png" alt=""><a href="http://star.mit.edu/cellbio/support.html"
+                                                          role="link" aria-label="Funding" target="_blank"><span
+        aria-hidden="true" class="scb_s_instructor_homepage_footer_support"
+        src="images/homepage/support.png">Funding</span></a>
+</div>
+</div>
+</div>
```

Comment:
something weird is happening here, we need indentation or to remove bonus divs...without using my text editor I don't know which.